specifications/cdx-format/openwayback-cdxj/index.md

```diff
@@ -199,7 +199,7 @@ is designed to facilitate this. A non-sorted CDXJ file is still valid as far as
 searching. 
 
 When sorted, the sorting should be done based on the native byte values of the characters. This is equivalent to using the GNU sort with 
-the collation settings `LC_ALL=C`.
+the collation settings `LC_COLLATION=C`.
```

Comment:
The libc doc linked in the description seems to mention LC_COLLATE not LC_COLLATION.

Reply:
Thanks. I've updated that.